src/server/util/BasicOnErrorHttpHandler.ts

```diff
@@ -0,0 +1,14 @@
+import { getLoggerFor } from '../../logging/LogUtil';
+import type { HttpHandlerInput } from '../../server/HttpHandler';
+import { OnErrorHttpHandler } from '../../server/HttpHandler';
+
+export class BasicOnErrorHttpHandler extends OnErrorHttpHandler {
+  private readonly logger = getLoggerFor(this);
+
+  public async handle(input: { error: unknown; input: HttpHandlerInput }): Promise<void> {
+    if (input.error instanceof Error) {
+      input.input.response.end(`${input.error.stack}`);
+    }
+    input.input.response.end('Error');
+  }
```

Comment:
This does not seem right. Doesn't the server do this by default? @joachimvh

Reply:
It does, but only if you're at the end of the waterfall handler. This catches errors from within the specially made `ErrorHandlingWaterfallHandler`